style.css

```diff
@@ -0,0 +1,37 @@
+#main{
+    align-items: center;
+    background-color: rgb(184, 179, 179);
+    display: flex;
+    flex-direction: column ;
+}
```

Comment:
- It would be better if you give margin and padding zero at the beginning of your CSS as it will make your structure clean. Kindly also add a box-sizing border-box which will make the look and feel of the website good.

Reply:
Great job being polite, phrasing your comment as a suggestion and giving a reason for the requested change.